src/librustc/ty/maps/plumbing.rs

```diff
@@ -193,7 +205,7 @@ macro_rules! define_maps {
 
         define_map_struct! {
             tcx: $tcx,
-            input: ($(([$($modifiers)*] [$($attr)*] [$name]))*)
+            input: ($(([$($attr)*] [$name]))*)
```

Comment:
I had to remove this use of `$modifiers` otherwise I got this error:
```
error: no rules expected the token `nocycle`
   --> librustc\ty\maps\mod.rs:270:6
    |
270 |     [nocycle] fn is_panic_runtime: IsPanicRuntime(CrateNum) -> bool,
    |      ^^^^^^^

error: aborting due to previous error
```
I'm not sure why this happens. `define_map_struct` should accept any tokens there. 😕

Reply:
see my comment below :)